web/templates/settings/notification_settings.hbs

```diff
@@ -110,7 +110,12 @@
         <div class="desktop-notification-settings-banners"></div>
 
         {{#unless for_realm_settings}}
-        <p><a class="send_test_notification">{{t "Send a test notification" }}</a></p>
+        {{> ../components/action_button
```

Comment:
This change means that there is no margin below the banner shown when browser does not allow notifcations.